lms/views/error.py

```diff
@@ -1,48 +1,66 @@
 import h_pyramid_sentry
 from pyramid import httpexceptions, i18n
-from pyramid.view import forbidden_view_config, notfound_view_config
+from pyramid.view import (
+    exception_view_config,
+    forbidden_view_config,
+    notfound_view_config,
+)
 
 from lms.validation import ValidationError
 
 _ = i18n.TranslationStringFactory(__package__)
 
+DEFAULT_RENDERER = "lms:templates/error.html.jinja2"
 
-@notfound_view_config(renderer="lms:templates/error.html.jinja2")
-def notfound(request):
+
+def _http_error(exc, request):
+    """Handle an HTTP 4xx or 5xx exception."""
+    request.response.status_int = exc.status_int
+    return {"message": str(exc)}
+
+
+@notfound_view_config(renderer=DEFAULT_RENDERER)
+def notfound(_exc, request):
     request.response.status_int = 404
     return {"message": _("Page not found")}
 
 
-@forbidden_view_config(renderer="lms:templates/error.html.jinja2")
-def forbidden(request):
+@forbidden_view_config(renderer=DEFAULT_RENDERER)
+def forbidden(_exc, request):
```

Comment:
The `_exc` param doesn't do anything here, but when debugging it contains the exception and therefore stack trace of what went wrong, which is extremely helpful.